impl/src/test/groovy/com/okta/authn/sdk/impl/resource/BulkResourceTest.groovy

```diff
@@ -205,7 +206,8 @@ class BulkResourceTest {
                 resource.setProperty(property, "2001-07-04T12:08:56.235-0700")
             }
             else if (property instanceof MapProperty) {
-                resource.setProperty(property, [one: "two"])
+                String jsonStr = "{\"one\":\"two\",\"factors\":{\"_embedded\":{\"challenge\":{\"correctAnswer\":42}}}}";
```

Comment:
Doesn't look like the tests is making any new validations for the new functionality. Do we need to add more validations?